Magento2/Sniffs/Commenting/ClassAndInterfacePHPDocFormattingSniff.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+/**
+ * Copyright © Magento. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento2\Sniffs\Commenting;
+
+use PHP_CodeSniffer\Sniffs\Sniff;
+use PHP_CodeSniffer\Files\File;
+
+/**
+ * Detects PHPDoc formatting for classes and interfaces.
+ */
+class ClassAndInterfacePHPDocFormattingSniff implements Sniff
+{
+    /**
+     * @var PHPDocFormattingValidator
+     */
+    private $PHPDocFormattingValidator;
+
+    /**
+     * @var string[] List of tags that can not be used in comments
+     */
+    private $forbiddenTags = [
+        '@author',
```

Comment:
Why `@author` is forbidden? As these tests designed to be used not only by Magento - it have to be allowed

Reply:
Agree, let's make `$forbiddenTags` public and remove `@author` from the default array. For public properties we can pass values through `ruleset.xml`. As an option, for the core ruleset we will be able to overwrite those values. More details: https://github.com/squizlabs/PHP_CodeSniffer/wiki/Customisable-Sniff-Properties

Reply:
Done as agreed. Override through `rulset.xml` works as expected